public/styles/nav.css

```diff
@@ -0,0 +1,37 @@
+.container-nav-bar {
```

Comment:
Great, I like the idea of having modular css files that correspond with each major component. I've never done it like this however, so I'm not really sure if the pitfalls associated with this approach. What are your thoughts?

Also, I thought I saw a css loader in your webpack config?